extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -10,7 +10,8 @@ var React = require('react'),
     Skin = require('./skin'),
     SkinJSON = require('../config/skin'),
     Bulk = require('bulk-require'),
-    Localization = Bulk('./config', ['languageFiles/*.json']);
+    Localization = Bulk('./config', ['languageFiles/*.json']),
+    Cookies = require('js-cookie');
 
 OO.plugin("Html5Skin", function (OO, _, $, W) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "W". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)